composer.json

```
@@ -16,7 +16,8 @@
   },
   "require": {
     "php": "^7.4 || ^8",
-    "illuminate/support": "^8 || ^9",
+    "bensampo/laravel-enum": "^5.3",
```

```suggestion
    "bensampo/laravel-enum": "^2 || ^3 || ^4 || ^5",
```

Necessary to match illuminate/support